web-app/public/locales/en/feeds.json

```diff
@@ -1,19 +1,19 @@
 {
     "feeds": "Feeds",
-    "dataType": "Data Type",
+    "dataType": "Data Format",
```

Comment:
If we want to change this to "Data Format" it will be the default standard everywhere: Feed page, submission form, search table, etc. Is this going to be the terminology going forward or just a one-off?

Reply:
Terminology going forward